packages/itwinui-css/backstop/assets/demo.css

```diff
@@ -1,32 +1,40 @@
 @font-face {
-  font-family: 'Open Sans';
+  font-family: 'Noto Sans Mono';
+  src: url('fonts/NotoSansMono-VariableFont_wdth\,wght.ttf') format('truetype');
+}
+@font-face {
+  font-family: 'Noto Sans';
   font-style: normal;
   font-weight: 300;
-  src: url('fonts/open-sans-v28-latin-300.woff2') format('woff2');
+  src: url('fonts/NotoSans-Light.ttf') format('truetype');
 }
+
 @font-face {
-  font-family: 'Open Sans';
+  font-family: 'Noto Sans';
   font-style: normal;
   font-weight: 400;
-  src: url('fonts/open-sans-v28-latin-regular.woff2') format('woff2');
+  src: url('fonts/NotoSans-Regular.ttf') format('truetype');
```

Comment:
Why `ttf`? We should be using `woff2`.

Reply:
Google fonts doesn't have woff2? https://fonts.google.com/noto/specimen/Noto+Sans

Reply:
I added Noto Sans woff2 files. Noto sans mono is variable font - maybe we can leave it as is?

Reply:
Noto Sans Mono size:
- variable ttf: 1.3MB
- variable woff2 latin full: 51KB
- variable woff2 latin weights: 28KB
- weight 400 woff2 latin: 9KB

I would be ok with any of the latter three. 1.3MB is not an acceptable size for any webfont.

Reply:
Changed to variable woff2 latin full: 51KB